### PR TITLE
clarify citation info in doc

### PR DESCRIPTION
**Pull request by eclare108213**
- [x] Short (1 sentence) summary of your PR: 
Modifying information in the documentation about how to cite the code  (see [CICE issue #465](https://github.com/CICE-Consortium/CICE/issues/465))
- [x] Developer(s): 
    E. Hunke
- [x] Suggest PR reviewers from list in the column to the right.
- [ ] Please copy the PR test results link or provide a summary of testing completed below.
    No changes to the code base, only documentation. 
- How much do the PR code changes differ from the unmodified code? 
    - [x] bit for bit
    - [ ] different at roundoff level
    - [ ] more substantial 
- Does this PR create or have dependencies on CICE or any other models?
    - [ ] Yes
    - [x] No
- Does this PR add any new test cases?
    - [ ] Yes
    - [x] No
- Is the documentation being updated? ("Documentation" includes information on the wiki or in the .rst files from doc/source/, which are used to create the online technical docs at https://readthedocs.org/projects/cice-consortium-cice/.)
    - [x] Yes
    - [ ] No, does the documentation need to be updated at a later time?
        - [ ] Yes
        - [ ] No 
- [x] Please provide any additional information or relevant details below:

Similar verbiage for CICE will be added.  We also need to update the wiki Version Index to point to the documentation rather than zenodo, for citation instructions.

### CODE REVIEW

File: doc/source/intro/citing.rst [hunk 1]
```diff
@@ -5,14 +5,27 @@
...
-However, each released version of Icepack will also receive its own, unique DOI that can be 
-used for citations as well.
+If you use Icepack, please cite the version number of the code you
+are using or modifying.
 
-Please also make the CICE Consortium aware of any publications and model use.
+If using code from the repository master that includes modifications
+that have not yet been released with a version number, then in
+addition to the most recent version number, the hash at time of
+download can be cited, determined by executing the command ‘git log’
```
Comment (phil-blain) [inline doc/source/intro/citing.rst:22]:
```suggestion
download can be cited, determined by executing the command ``git log``
```

File: doc/source/intro/citing.rst [hunk 2]
```diff
@@ -5,14 +5,27 @@
...
-DOI:10.5281/zenodo.1213462 (https://zenodo.org/record/1213462) 
 
-This DOI can be used to cite all Icepack versions and the URL will default to the most recent version. 
-However, each released version of Icepack will also receive its own, unique DOI that can be 
-used for citations as well.
+If you use Icepack, please cite the version number of the code you
+are using or modifying.
 
-Please also make the CICE Consortium aware of any publications and model use.
+If using code from the repository master that includes modifications
```
Comment (phil-blain) [inline doc/source/intro/citing.rst:19]:
```suggestion
If using code from the CICE-Consortium repository ``master`` branch that includes modifications
```

File: doc/source/intro/citing.rst [hunk 3]
```diff
@@ -5,14 +5,27 @@
...
+that have not yet been released with a version number, then in
+addition to the most recent version number, the hash at time of
+download can be cited, determined by executing the command ‘git log’
+in your clone. The last 7 digits are generally sufficient to identify
+the latest commit.
```
Comment (phil-blain) [inline doc/source/intro/citing.rst:24]:
```suggestion
in your clone. 
```

I think it's better to cite the full hash. (And Git will find stuff with the *first* digits if they are unique, not the last).

**Comment by eclare108213** [comment]
https://github.com/CICE-Consortium/CICE/pull/487 for CICE has been merged.  This PR for Icepack should be equivalent.  Someone please review it so we can merge it too.  Thanks!

**Pull request merged**